lib/flv/transmuxer.js

```diff
@@ -442,11 +447,9 @@ Transmuxer = function(options) {
     .pipe(this.metadataStream)
     .pipe(coalesceStream);
   // if CEA-708 parsing is available, hook up a caption stream
-  if (muxjs.mp2t.CaptionStream) {
-    captionStream = new muxjs.mp2t.CaptionStream();
-    h264Stream.pipe(captionStream)
-      .pipe(coalesceStream);
-  }
+  captionStream = new m2ts.CaptionStream();
```

Comment:
With require this check didn't make sense (either you require it or you don't). Also there is little compelling reason to *not* include caption parsing support.